.gitignore

```diff
@@ -1,13 +1,14 @@
 *~
 
+.vagrant
+/.config.args
 /Makefile
+/acbuild
 /autom4te.cache
+/build-rir*
+/build-rkt*
 /config.log
 /config.status
-/.config.args
 /configure
-/build-rkt*
-/build-rir*
-.vagrant
+/dist
```

Comment:
Gah, dumb oversight on my part! I was thinking of a different project. Can you please fix?